lib/org/chromium/webidl/FetchSpecRunner.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'FetchSpecRunner',
+  extends: 'org.chromium.webidl.PipelineRunner',
+
+  requires: [
+    'foam.net.RetryHTTPRequest',
+    'org.chromium.webidl.HTMLFileContents'
+  ],
+
+  documentation: `A runnable box that expects to receive a PipelineMessage
+    containing an array of URLs. The component will spawn a HTTP worker,
+    which fetches the pages at the given URLs.`,
+
+  constants: {
+    MISSING_URLS: 'URL property of PipelineMessage is missing!',
```

Comment:
This looks like it's only used in one place (not shared), and we haven't established practices for constantizing error messages. We can probably just use this *in situ* (no constant required). Let's have something like `FetchSpecRunner: No URLs found on input message`.